src/components/BlogContent.jsx

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+
+const BlogContent = (props) => {
+  return (
+    <div className="shadow-md w-80 lg:w-65">
+      <a>
+        <div className="h-80 lg:h-65 relative">
+          <h6 className="text-[#FFFFFF] font-bold text-sm bg-[#E74040] absolute ml-3 mt-3 px-3 py-1">
+            NEW
+          </h6>
+          <img src={props.blog.image} className="w-full h-full object-cover " />
+        </div>
+
+        <div className="m-7 text-left">
+          <p className="text-[#737373] text-xs font-semibold w-40 flex flex-row justify-between">
```

Comment:
İki kez aynı text-[#737373] ve font-semibold kullanılmış. Bu tekrarı azaltmak ve CSS'in yeniden kullanılabilirliğini artırmak için genel bir CSS sınıfı oluşturulabilir.